en/faq.md

```diff
@@ -0,0 +1,77 @@
+## FAQ
+
+### Why was the core of the SDK written in C++?
+
+The aim was to have an API in a language which is cross-platform and has many language bindings.
+Additionally, the library needs to be lightweight and fast, so it does not slow down for onboard usage at higher rate messaging.
+
+### Are multiple vehicles supported?
+
+Yes, the core C++ SDK is designed to support multiple vehicles. A vehicle is referred to a system in this SDK. The language wrappers so far only support one vehicle at a time. (However, nothing stops you from instantiating multiple copies of it to deal with multiple systems.)
```

Comment:
Modify to state that the C++ core does, but that the bindings do not

Reply:
It is now:
> Yes, the core C++ SDK is designed to support multiple vehicles. A vehicle is referred to a system in this SDK. The language wrappers so far only support one vehicle at a time. (However, nothing stops you from instantiating multiple copies of wrappers to deal with multiple systems.)

Is that what you meant?